protocols/floodsub/src/topic.rs

```diff
@@ -47,7 +47,7 @@ impl TopicHash {
 /// Built topic.
 #[derive(Debug, Clone)]
 pub struct Topic {
-    descriptor: rpc_proto::TopicDescriptor,
+    pub descriptor: rpc_proto::TopicDescriptor,
```

Comment:
Docs on public field

Reply:
Also this shouldn't be public. There should be a getter like `name(&self) -> &str`.
If we stop using hashes altogether, then the whole content of the module may need some revisiting.

Reply:
OK, will implement a getter tomorrow in Sydney.